athlinks-schema.ttl

```diff
@@ -195,81 +175,59 @@ athlinks:agePercentage
   rdfs:domain athlinks:IntervalResult;
   rdfs:range xsd:decimal .
 
-athlinks:isHidden
-  a owl:DataTypeProperty ;
-  rdfs:domain athlinks:AthleticRace;
-  rdfs:range xsd:boolean .
-
-athlinks:isVisible
-  a owl:DataTypeProperty ;
-  rdfs:domain athlinks:AthleticRace;
-  rdfs:range xsd:boolean .
-
-athlinks:isEnabled
-  a owl:DataTypeProperty ;
-  rdfs:domain athlinks:AthleticRace;
-  rdfs:range xsd:boolean .
-
-athlinks:isApproved
-  a owl:DataTypeProperty ;
-  rdfs:domain athlinks:AthleticRace;
-  rdfs:range xsd:boolean .
-
-athlinks:isHandicap
-  a owl:DataTypeProperty ;
-  rdfs:domain athlinks:AthleticRace;
-  rdfs:range xsd:boolean .
-
-athlinks:isNavigable
-  a owl:DataTypeProperty ;
-  rdfs:domain athlinks:AthleticRace;
-  rdfs:range xsd:boolean .
-
-athlinks:elevUnit
-  a owl:DataTypeProperty ;
-  rdfs:domain athlinks:TerrainSpecifics;
-  rdfs:range xsd:float .
-
-athlinks:altitude
-  a owl:DataTypeProperty ;
-  rdfs:domain athlinks:TerrainSpecifics;
-  rdfs:range xsd:float .
-
-athlinks:startMarked
-  a owl:DataTypeProperty ;
+athlinks:elevation a owl:DataTypeProperty;
+  rdfs:label "Elevation";
+  rdfs:comment "Elevation";
+  rdfs:domain schema:Landform;
+  rdfs:range schema:QuantitativeValue.
+
+athlinks:altitude a owl:DataTypeProperty;
+  rdfs:label "Altitude";
+  rdfs:comment "Altitude";
+  rdfs:domain schema:Landform;
+  rdfs:range schema:QuantitativeValue.
+
+athlinks:elevationGradeType a owl:DataTypeProperty;
+  rdfs:label "Elevation Grade";
+  rdfs:comment "Elevation Grade";
+  rdfs:domain schema:Landform;
+  rdfs:range schema:QuantitiveValue.
+
+athlinks:elevationType a owl:DataTypeProperty;
+  rdfs:label "Elevation";
+  rdfs:comment "Elevation";
+  rdfs:domain schema:Landform;
+  rdfs:range schema:QuantitiveValue.
```

Comment:
Whats distinct about athlinks:elevationType vs athlinks:elevation? They both have the same label and description.